moveit_core/collision_detection/include/moveit/collision_detection/collision_common.h

```diff
@@ -273,14 +280,14 @@ struct DistanceRequest
   /// The group name
   std::string group_name;
 
-  /// The set of active components to check
+  /// The set of active components to check. Active refers to components which move.
```

Comment:
I think, active components could be anything. However, they should coincide with links that move.